### PR TITLE
Created my first pull request

### PR DESCRIPTION
**Pull request by tiko-star**
I have added new php file and updated readme file!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# geek-course
\ No newline at end of file
+# geek-course
```
Comment (tiko-star) [inline README.md]: rename geek-course to Geek Course

File: hello.php [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<?php
+
+echo "Hello World";
```
Comment (tiko-star) [inline hello.php]: add this sign **"!"**

Comment (tiko-star) [reply hello.php]: done!

File: README.md [hunk 3]
```diff
@@ -1 +1,3 @@
-# geek-course
\ No newline at end of file
+# Geek-Course
+
+# This course is designed for learning purposes!
```
Comment (Asilbekyan) [inline README.md]: Change this

File: hello.php [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<?php
+
+echo "Hello World!";
```
Comment (Asilbekyan) [inline hello.php]: change Hello to Good Bye

**Review by Asilbekyan — COMMENTED** [review]
Please change something

File: README.md (hunk 3 above)
Comment (tiko-star) [reply README.md]: fixed

File: hello.php (hunk 4 above)
Comment (tiko-star) [reply hello.php]: fixed

**Review by Asilbekyan — APPROVED** [review]

**Pull request merged**